### DISCUSSION TITLE
skywalking_log-* index  on es can be more readable

### DISCUSSION THREAD

**Discussion by sinlang** [apache/skywalking#9403] [Q&A]
![image](https://user-images.githubusercontent.com/4089994/181687273-b0694eb6-899f-4adf-9d5e-9d42cba4d80e.png)

expand all tags in tags_raw_data, show service_instance_id as it is

**Comment by sinlang** [comment]
i don't known why use base64 here?

**Comment by wu-sheng** [comment]
Why you want to read them from database? This is not project design.If you need any data, read through official GraphQL APIs.
